beacon-chain/blockchain/receive_block.go

```diff
@@ -134,6 +137,8 @@ func (s *Service) ReceiveBlockNoPubsubForkchoice(ctx context.Context, block *eth
 	// Reports on block and fork choice metrics.
 	s.reportSlotMetrics(block.Slot)
 
+	// We write the latest saved head root to a feed for consumption by other services.
+	s.headUpdatedFeed.Send(root)
```

Comment:
Used here @nisdas and imported by the archiver service

Reply:
missed this, thanks